### PR TITLE
Fixed the fogbugz case 1318629

### PR DESCRIPTION
**Pull request by thomas-zeng**
---
### Purpose of this PR
Fixed the fogbugz case https://fogbugz.unity3d.com/f/cases/1318629/

---
### Testing status
Tested locally with the repro project.

---
### Comments to reviewers
Moved base camera `BeginCameraRendering` before InitializeCameraData

### CODE REVIEW

File: com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs [hunk 1]
```diff
@@ -557,6 +553,10 @@ static void RenderCameraStack(ScriptableRenderContext context, Camera baseCamera
                 }
             }
 
+            using (new ProfilingScope(null, Profiling.Pipeline.endCameraRendering))
+            {
+                EndCameraRendering(context, baseCamera);
+            }
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs:559]: This doesn't look correct. EndCameraRendering for overlay cameras is called before the EndCameraRendering for base camera.

**Review by phi-lira — CHANGES_REQUESTED** [review]
Can you describe in the PR more about why the EndCameraRendering for base camera has to be called after after all EndCameraRendering of overlay cameras? That seems inconsistent and most likely will introduce issues for users.

**Review by erikabar — APPROVED** [review]
Developer tested with repro project, approving

File: com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs (hunk 1 above)
Comment (thomas-zeng) [reply com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs]: Addressed in f1a61a9

**Review by phi-lira — APPROVED** [review]

**Comment by phi-lira** [comment]
Lots of failed tests, maybe remerge master to see the status. Currently one Hybrid tests should be failing.

**Comment by thomas-zeng** [comment]
Hi @phi-lira,

I have merged with master branch and rerun yamato tests. The test results are all green besides hybrid renderer tests.

**Review by sandy-carter-unity — APPROVED** [review]

**Pull request merged**